shared/js/background/chrome-events.es6.js

```diff
@@ -857,8 +857,6 @@ chrome.alarms.onAlarm.addListener(alarmEvent => {
             .catch(e => console.log(e))
     } else if (alarmEvent.name === 'clearExpiredHTTPSServiceCache') {
         httpsService.clearExpiredCache()
-    } else if (alarmEvent.name === 'updateUserAgentData') {
-        // TODO: Reinstate user agent rotation
```

Comment:
we also need to remove corresponding alarm initialization - https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/46c81ba7b4fe57c520af29b9a77cea6dd22f1655/shared/js/background/chrome-events.es6.js#L835